<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1152</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QWidget" name="wgtControls" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tbControls">
          <attribute name="title">
           <string>Scan</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="title">
              <string>Controls</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <property name="leftMargin">
               <number>2</number>
              </property>
              <property name="topMargin">
               <number>2</number>
              </property>
              <property name="rightMargin">
               <number>2</number>
              </property>
              <property name="bottomMargin">
               <number>2</number>
              </property>
              <item>
               <widget class="QPushButton" name="cmdStartStop">
                <property name="text">
                 <string>Start Scanning</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="grpSweepFreqControls">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="title">
              <string>Scanning Frequencies</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout">
              <property name="spacing">
               <number>2</number>
              </property>
              <property name="leftMargin">
               <number>2</number>
              </property>
              <property name="topMargin">
               <number>2</number>
              </property>
              <property name="rightMargin">
               <number>2</number>
              </property>
              <property name="bottomMargin">
               <number>2</number>
              </property>
              <item>
               <widget class="QWidget" name="wgtFreqMin" native="true">
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <property name="spacing">
                  <number>2</number>
                 </property>
                 <property name="leftMargin">
                  <number>2</number>
                 </property>
                 <property name="topMargin">
                  <number>2</number>
                 </property>
                 <property name="rightMargin">
                  <number>2</number>
                 </property>
                 <property name="bottomMargin">
                  <number>2</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="lblFreqMin">
                   <property name="text">
                    <string>Min (MHz)</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QDoubleSpinBox" name="spnFreqLow">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>48</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <pointsize>11</pointsize>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">QDoubleSpinBox::up-button { width: 32px; }
QDoubleSpinBox::down-button { width: 32px; }</string>
                   </property>
                   <property name="decimals">
                    <number>2</number>
                   </property>
                   <property name="maximum">
                    <double>8000.000000000000000</double>
                   </property>
                   <property name="value">
                    <double>800.000000000000000</double>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="wgtFreqMax" native="true">
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <property name="spacing">
                  <number>2</number>
                 </property>
                 <property name="leftMargin">
                  <number>2</number>
                 </property>
                 <property name="topMargin">
                  <number>2</number>
                 </property>
                 <property name="rightMargin">
                  <number>2</number>
                 </property>
                 <property name="bottomMargin">
                  <number>2</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="lblFreqMax">
                   <property name="text">
                    <string>Max (MHz)</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QDoubleSpinBox" name="spnFreqHigh">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>48</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">QDoubleSpinBox::up-button { width: 32px; }
QDoubleSpinBox::down-button { width: 32px; }</string>
                   </property>
                   <property name="maximum">
                    <double>9000.000000000000000</double>
                   </property>
                   <property name="value">
                    <double>900.000000000000000</double>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="wgtRBW" native="true">
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <property name="spacing">
                  <number>2</number>
                 </property>
                 <property name="leftMargin">
                  <number>2</number>
                 </property>
                 <property name="topMargin">
                  <number>2</number>
                 </property>
                 <property name="rightMargin">
                  <number>2</number>
                 </property>
                 <property name="bottomMargin">
                  <number>2</number>
                 </property>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="wgtBins" native="true">
                <layout class="QVBoxLayout" name="verticalLayout_12">
                 <property name="leftMargin">
                  <number>2</number>
                 </property>
                 <property name="topMargin">
                  <number>2</number>
                 </property>
                 <property name="rightMargin">
                  <number>2</number>
                 </property>
                 <property name="bottomMargin">
                  <number>2</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="lblBins">
                   <property name="text">
                    <string>Number of Bins</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSlider" name="sldBins">
                   <property name="minimum">
                    <number>6</number>
                   </property>
                   <property name="maximum">
                    <number>15</number>
                   </property>
                   <property name="value">
                    <number>10</number>
                   </property>
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="tickPosition">
                    <enum>QSlider::TicksBelow</enum>
                   </property>
                   <property name="tickInterval">
                    <number>1</number>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="lblRBW">
                   <property name="text">
                    <string>RBW: </string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="lblBinperscan">
                   <property name="text">
                    <string>Bins per scan :</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="lblScanOverlap">
                <property name="text">
                 <string>Scan overlap (0-25)%</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="sldOverlap">
                <property name="maximum">
                 <number>25</number>
                </property>
                <property name="value">
                 <number>10</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>Radio Settings</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_10">
              <property name="spacing">
               <number>2</number>
              </property>
              <property name="leftMargin">
               <number>2</number>
              </property>
              <property name="topMargin">
               <number>2</number>
              </property>
              <property name="rightMargin">
               <number>2</number>
              </property>
              <property name="bottomMargin">
               <number>2</number>
              </property>
              <item>
               <widget class="QWidget" name="wgtAntenna" native="true">
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <property name="spacing">
                  <number>2</number>
                 </property>
                 <property name="leftMargin">
                  <number>2</number>
                 </property>
                 <property name="topMargin">
                  <number>2</number>
                 </property>
                 <property name="rightMargin">
                  <number>2</number>
                 </property>
                 <property name="bottomMargin">
                  <number>2</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>Antenna</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="cmbAntenna"/>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="widget_2" native="true">
                <layout class="QVBoxLayout" name="verticalLayout_11">
                 <property name="spacing">
                  <number>2</number>
                 </property>
                 <property name="leftMargin">
                  <number>2</number>
                 </property>
                 <property name="topMargin">
                  <number>2</number>
                 </property>
                 <property name="rightMargin">
                  <number>2</number>
                 </property>
                 <property name="bottomMargin">
                  <number>2</number>
                 </property>
                 <item>
                  <widget class="QCheckBox" name="chkAGC">
                   <property name="text">
                    <string>Auto Gain Mode</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="wgtGain" native="true">
                <layout class="QHBoxLayout" name="horizontalLayout_6">
                 <property name="spacing">
                  <number>2</number>
                 </property>
                 <property name="leftMargin">
                  <number>2</number>
                 </property>
                 <property name="topMargin">
                  <number>2</number>
                 </property>
                 <property name="rightMargin">
                  <number>2</number>
                 </property>
                 <property name="bottomMargin">
                  <number>2</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="lblGain">
                   <property name="text">
                    <string>Gain</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="cmbGain"/>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="sldGain">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tbDetect">
          <attribute name="title">
           <string>Detect</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_14">
           <item>
            <widget class="QWidget" name="widget_3" native="true">
             <layout class="QVBoxLayout" name="verticalLayout_13">
              <item>
               <widget class="QComboBox" name="comboBox">
                <item>
                 <property name="text">
                  <string>Peak Detector</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Straight Line Threshold</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Noise Riding Threshold</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Spectrogram Energy </string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>207</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>256</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tbViewOpt">
          <attribute name="title">
           <string>Options</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <property name="spacing">
            <number>2</number>
           </property>
           <property name="leftMargin">
            <number>2</number>
           </property>
           <property name="topMargin">
            <number>2</number>
           </property>
           <property name="rightMargin">
            <number>2</number>
           </property>
           <property name="bottomMargin">
            <number>2</number>
           </property>
           <item>
            <widget class="QWidget" name="widget" native="true">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <property name="spacing">
               <number>2</number>
              </property>
              <property name="leftMargin">
               <number>2</number>
              </property>
              <property name="topMargin">
               <number>2</number>
              </property>
              <property name="rightMargin">
               <number>2</number>
              </property>
              <property name="bottomMargin">
               <number>2</number>
              </property>
              <item>
               <widget class="QCheckBox" name="chkFFT">
                <property name="text">
                 <string>FFT</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="chkWaterfall">
                <property name="text">
                 <string>Waterfall</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="chkMarkers">
                <property name="text">
                 <string>Markers</string>
                </property>
                <property name="checked">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="grpTraceSettings">
                <property name="title">
                 <string>Trace Settings</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_8">
                 <property name="spacing">
                  <number>4</number>
                 </property>
                 <property name="leftMargin">
                  <number>2</number>
                 </property>
                 <property name="topMargin">
                  <number>2</number>
                 </property>
                 <property name="rightMargin">
                  <number>2</number>
                 </property>
                 <property name="bottomMargin">
                  <number>2</number>
                 </property>
                 <item>
                  <widget class="QPushButton" name="cmdClearMax">
                   <property name="text">
                    <string>Clear Max</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QCheckBox" name="chkShowMax">
                   <property name="text">
                    <string>Show Max Trace</string>
                   </property>
                   <property name="checked">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QFrame" name="frmAveraging">
                   <property name="frameShape">
                    <enum>QFrame::StyledPanel</enum>
                   </property>
                   <property name="frameShadow">
                    <enum>QFrame::Raised</enum>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_9">
                    <item>
                     <widget class="QLabel" name="lblFFTAverage">
                      <property name="text">
                       <string>FFT Averaging</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QSlider" name="sldAverage">
                      <property name="styleSheet">
                       <string notr="true"/>
                      </property>
                      <property name="minimum">
                       <number>1</number>
                      </property>
                      <property name="maximum">
                       <number>20</number>
                      </property>
                      <property name="value">
                       <number>10</number>
                      </property>
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                      <property name="tickPosition">
                       <enum>QSlider::TicksBelow</enum>
                      </property>
                      <property name="tickInterval">
                       <number>2</number>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="wgtGraphBase" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>2</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="wgt_FFT" name="wgtfft" native="true"/>
       </item>
       <item>
        <widget class="wgt_waterfall" name="wgtwaterfall" native="true"/>
       </item>
       <item>
        <widget class="wgt_marker_table" name="wgtMarkers" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1152</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionRadio"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionRadio">
   <property name="text">
    <string>Radio</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>wgt_FFT</class>
   <extends>QWidget</extends>
   <header>wgt_fft.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>wgt_waterfall</class>
   <extends>QWidget</extends>
   <header>wgt_waterfall.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>wgt_marker_table</class>
   <extends>QWidget</extends>
   <header>wgt_marker_table.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
